contracts/SafePolicyGuard.sol

```diff
@@ -15,8 +15,21 @@ import {ISafe} from "./interfaces/ISafe.sol";
 contract SafePolicyGuard is PolicyEngine, ISafeModuleGuard, ISafeTransactionGuard {
     using AccessSelector for AccessSelector.T;
 
-    // keccak256("guard_manager.guard.address")
-    bytes32 internal constant _GUARD_STORAGE_SLOT = 0x4a204f620c8c5ccdca3fd54d003badd85ba500436a431f0cbda4f558c93c34c8;
+    /**
+     * @notice The configuration data struct for a policy.
+     * @custom:member target The target address for the policy.
+     * @custom:member selector The selector for the policy.
+     * @custom:member operation The operation for the policy.
+     * @custom:member policy The policy address.
+     * @custom:member data The data for the policy.
+     */
+    struct Configuration {
+        address target;
+        bytes4 selector;
+        Operation operation;
```

Comment:
ubernit: 
```suggestion
        AccessSelector.T access;
```

Also... we need a better name than `AccessSelector` :joy:

Reply:
Will keep it for now, though will think of new names 😅